### PR TITLE
workflow action partial

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

File: resources/partial/workflow-action-springboot/${id}-workflow-action.client-extension-config.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (gamerson) [inline resources/partial/workflow-action-springboot/${id}-workflow-action.client-extension-config.json:1]: So i guess we need to have a followup task for pushing this to gradle task and generalize that pattern.

File: resources/partial/workflow-action-springboot/client-extension.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+${id}-user-agent:
+  type: oauthApplicationUserAgent
+  scopes:
+    - Liferay.Headless.Admin.Workflow.everything
+
+assemble:
```
Comment (gamerson) [inline resources/partial/workflow-action-springboot/client-extension.yaml:6]: if they already have assemble, our partials will not merge.  but we can fix `by and by`.

**Review by gamerson — APPROVED** [review]

File: resources/partial/workflow-action-springboot/${id}-workflow-action.client-extension-config.json (hunk 1 above)
Comment (rotty3000) [reply resources/partial/workflow-action-springboot/${id}-workflow-action.client-extension-config.json:1]: yes!

File: resources/partial/workflow-action-springboot/client-extension.yaml (hunk 2 above)
Comment (rotty3000) [reply resources/partial/workflow-action-springboot/client-extension.yaml:6]: agreed!

**Pull request merged**